OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -68,11 +68,10 @@ COPY $OHS_PKG install.file oraInst.loc /u01/
 RUN yum install -y \
         unzip tar nfs-utils binutils compat-libcap1 compat-libstdc++ \
```

Comment:
I just noticed `nfs-tools` here and I'm still not convinced about why you need `ksh`. Just pointing to the documentation is not sufficient. You need to evaluate all the requirements to determine if they're still valid, because anything you add will make the image larger.